.vscode/settings.json

```diff
@@ -7,4 +7,5 @@
         "doc/doxygen/xml/": true,
         "doc/sphinx/_build/": true
     },
+    "C_Cpp.errorSquiggles": "Disabled",
```

Comment:
Please remove this.

Reply:
Are you sure? squiggles are not working in our pagmo setup where headers are searched with the ```<>``` symbol and not ```""```. Without this I get consistently error from the c++ extension I use and I need to activate it each time.

Reply:
Perhaps then you need a CMake extensions that is able to pick up the extra include paths from our build system?

Or maybe it's just better to remove this file entirely. I just hate that it always picks up details form developer's custom setups.

Reply:
Actually, go ahead, get rid of it and add ``.vscode`` to the ``.gitignore``, so that we are free to configure our environments without stepping on each other's toes.

Reply:
ok agreed